### PR TITLE
Added Swerve drive objects to main

### PR DESCRIPTION
**Pull request by dracco1993**

### CODE REVIEW

File: config.txt [hunk 1]
```diff
@@ -24,10 +24,10 @@ SteerEncMaxBR = 4.901
 SteerEncMaxFL = 4.92
 SteerEncMaxFR = 4.914
 
-SteerEncOffsetFR = 3.10
-SteerEncOffsetFL = 3.85
-SteerEncOffsetBR = 2.2
-SteerEncOffsetBL = 3.56
+SteerEncOffsetFR = 1.07
```
Comment (dracco1993) [inline config.txt:27]: This is a number.

File: src/org/redalert1741/robotBase/input/EdgeDetect.java [hunk 2]
```diff
@@ -4,12 +4,12 @@
 {
 	private boolean state;
 	
-	EdgeDetect()
+	public EdgeDetect()
```
Comment (dracco1993) [inline src/org/redalert1741/robotBase/input/EdgeDetect.java:7]: Test.

**Review by dracco1993 — COMMENTED** [review]
Overall comment.

**Review by tricktrap — COMMENTED** [review]
Looks messy but ok to me! :+1:

**Comment by randomtestfive** [comment]
It exists

**Pull request merged**